src/utils/string.hpp

```diff
@@ -301,6 +301,26 @@ inline std::vector<std::string> RSplit(const std::string_view src, const std::st
   return res;
 }
 
+/**
+ * Get a first part of a string by `delimiter` into a string.
+ */
+inline std::string_view GetFirstPartOfSplit(const std::string_view src, const char delimiter) {
+  auto delimPos = src.find(delimiter);
```

Comment:
```suggestion
  size_t delimPos = src.find(delimiter);
```
Nothing against auto but increases compilation time and if it is such short type like size_t I think it is better to have it explicit